jekyll/explorer.md

```diff
@@ -85,7 +85,7 @@ For creating standard heatmaps, we assess the physical risk over the 30 year win
 | Annual Temperature | Annual mean annual temperature at the asset location | °C | -5 to 45 | 100km | N/A | N/A |
 | Annual Precipitation | Annual total precipitation at the asset location | mm | 0 to 5000 | 100km | N/A | N/A |
 | Extreme Precipitation | Annual probabilistic projections of number of days in a year where precipitation exceeds 51mm | Number of days | 0 to 365 | 100km | N/A | N/A |
-| Wildfire Burned Area Fraction | Annual fraction of land exposed to wildfire within 14km radius of the asset | Fraction | 0.0 to 1.0 | 14km | LOW: 0.0 to 0.000081; MEDIUM: 0.00008 to 0.01; HIGH: > 0.01 | 1.0 |
+| Wildfire Burned Area Fraction | Annual fraction of land exposed to wildfire within 6.9km radius of the asset | Fraction | 0.0 to 1.0 | 6.9km | LOW: 0.0 to 0.000081; MEDIUM: 0.00008 to 0.01; HIGH: > 0.01 | 1.0 |
```

Comment:
@jasSust is there a corresponding HTML change here, too?

Reply:
@bcwaldon Yes! Added -- thanks!